packages/datetime/src/dateTimePicker.tsx

```diff
@@ -100,7 +100,11 @@ export class DateTimePicker extends AbstractPureComponent2<IDateTimePickerProps,
                     onChange={this.handleDateChange}
                     value={value}
                 />
-                <TimePicker {...this.props.timePickerProps} onChange={this.handleTimeChange} value={value} />
+                <TimePicker
```

Comment:
this component is deprecated, please revert these changes. users should be using `<DatePicker>` instead.

Reply:
Without this change, the test `changing the time before selecting a date works as expected` fails because clearing the date also clears the time because the `TimePicker` component now responds to being passed `null`, whereas before it was a no-op. Should I still revert this change?

Reply:
Ok, fair enough, we can include this change in the PR.